webpack.config.js

```diff
@@ -163,12 +163,10 @@ const webpackConfig = ( mode ) => {
 				path: __dirname + '/dist/assets/js',
 				chunkFilename: '[name]-[chunkhash].js',
 				publicPath: '',
-				/**
-				 * If multiple webpack runtimes (from different compilations) are used on the same webpage,
-				 * there is a risk of conflicts of on-demand chunks in the global namespace.
-				 *
-				 * @see (@link https://webpack.js.org/configuration/output/#outputjsonpfunction)
-				 */
+				// If multiple webpack runtimes (from different compilations) are used on the
+				// same webpage, there is a risk of conflicts of on-demand chunks in the global
+				// namespace.
+				// See: https://webpack.js.org/configuration/output/#outputjsonpfunction.
```

Comment:
If this was seen as a doc block before, perhaps a multiline comment would be preferable?

Reply:
Oh, yeah, I guess so. I guess it's the extra `*` that makes it seen as a docblock.